io-engine/src/lvs/lvs_error.rs

```diff
@@ -86,6 +86,11 @@ pub enum Error {
         name: String,
         msg: String,
     },
+    #[snafu(display("failed to resize lvol {}", name))]
+    RepResize {
```

Comment:
Rename to `ReplicaResize` for more clarity?

Reply:
You might have done it already but please ensure error codes are converted to grpc...

Reply:
> Rename to `ReplicaResize` for more clarity?

All the existing codes above this are using `Rep` so I've kept it consistent with that.